transform/transformers/ImageTransformer.py

```diff
@@ -118,16 +118,18 @@ def cleanup(self):
         shutil.rmtree(os.path.join(self.path))
 
     def response_ok(self, res):
+        msg = "Returned from sdx-sequence"
+
         if res.status_code == 200:
-            self.logger.info("Returned from service", request_url=res.url, status=res.status_code)
+            self.logger.info("{}".format(msg), request_url=res.url, status=res.status_code)
```

Comment:
It's only a couple of locations so may be worth simply hardcoding in place rather than factoring out and `format`ing back in.